### PR TITLE
Design view manage product.

### PR DESCRIPTION
**Pull request by ducls-2223**

### CODE REVIEW

**Comment by ducls-2223** [comment]
ready

File: resources/views/admin/category_show.blade.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                                            <label for="selectable1"></label>
+                                        </div>
+                                    </th>
+                                    <th>{{ __('admin.category') }}</th>
+                                    <th>{{ __('admin.category_parent') }}</th>
+                                    <th>{{ __('admin.action') }}</th>
+                                </tr>
+                                </thead>
+                                <tbody>
+                                <tr>
```
Comment (huyenminnn) [inline resources/views/admin/category_show.blade.php]: indent

File: resources/views/admin/order_show.blade.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+@section('content')
+    <div class="page-container">
+        <div class="main-content">
+            <div class="container-fluid">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="table-overflow">
+                            <table id="dt-opt" class="table table-hover table-xl">
+                                <thead>
+                                <tr>
```
Comment (huyenminnn) [inline resources/views/admin/order_show.blade.php]: tương tụ

File: routes/web.php [hunk 3]
```diff
@@ -17,6 +17,14 @@
 Route::get('/all', 'ProductController@index')->name('products.index');
 Route::get('/categories/{category}', 'CategoryController@show')->name('categories.show');
 
+Route::prefix('admin')->group(function() {
```
Comment (huyenminnn) [inline routes/web.php]: convention

**Comment by ducls-2223** [comment]
ready

**Review by huyenminnn — APPROVED** [review]

**Review by tuannt-0910 — APPROVED** [review]

**Pull request merged**